packages/k8s-triliovault-operator/k8s-triliovault-operator.patch

```diff
@@ -0,0 +1,32 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/k8s-triliovault-operator/charts-original/Chart.yaml packages/k8s-triliovault-operator/charts/Chart.yaml
+--- packages/k8s-triliovault-operator/charts-original/Chart.yaml
++++ packages/k8s-triliovault-operator/charts/Chart.yaml
+@@ -1,6 +1,7 @@
+ apiVersion: v1
+ appVersion: v2.0.2
+-description: K8s-TrilioVault-Operator is an operator designed to manage the K8s-TrilioVault Application Lifecycle.
++description: K8s-TrilioVault-Operator is an operator designed to manage the K8s-TrilioVault
++  Application Lifecycle.
+ home: https://github.com/trilioData/k8s-triliovault-operator
+ icon: https://www.trilio.io/wp-content/uploads/2021/01/Trilio-2020-logo-RGB-gray-green.png
+ maintainers:
+@@ -10,3 +11,6 @@
+ sources:
+ - https://github.com/trilioData/k8s-triliovault-operator
+ version: v2.0.2
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: k8s-triliovault-operator
+diff -x '*.tgz' -x '*.lock' -uNr packages/k8s-triliovault-operator/charts-original/app-readme.md packages/k8s-triliovault-operator/charts/app-readme.md
+--- packages/k8s-triliovault-operator/charts-original/app-readme.md
++++ packages/k8s-triliovault-operator/charts/app-readme.md
+@@ -0,0 +1,4 @@
++# TrilioVault for Kubernetes
++
++[K8s-TrilioVault-Operator](https://trilio.io) is an operator designed to manage the K8s-TrilioVault Application 
++Lifecycle.
```

Comment:
These changes shouldn't be in the patch. Files in the overlay directory are added when generating the asset, so there is no need to include them in patch. Is possible you saved this in the patch accidentally by generating the assets and doing make patch again without a make clean to cleanup the changes generating the assets leave in your working directory.

Reply:
If we remove the patch file then this won't be a problem, if not then I will remove these unnecessary changes from patch file.

Reply:
The patch can't be removed for partner charts since you need the annotations.

Reply:
Got it. Added the changes to this PR.